content/en/getting_started/database_monitoring/_index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Getting Started with Database Monitoring
+kind: documentation
+further_reading:
+    - link: '/database_monitoring/'
+      tag: 'Documentation'
+      text: 'Database Monitoring'
+    - link: '/database_monitoring/troubleshooting/'
+      tag: 'Documentation'
+      text: 'Troubleshooting'
+    - link: '/blog/database-performance-monitoring-datadog/'
+      tag: 'Blog'
+      text: 'Database performance monitoring with Datadog'
+---
+## Overview
+
+Datadog Database Monitoring allows you to easily understand the health and performance of your databases, and quickly determine the root cause of any problems.
+
+In one place, you can view:
+
+* Host-level metrics
+* Explain plans
+* Historical query performance metrics
+
+Work through this guide to set up Datadog Database Monitoring on an example PostgreSQL database. Next, identify an expensive query, troubleshoot a slow query, and create a dashboard to view changes in query volume.
+
+## Setup
+
+### Prerequisites
+
+Before getting started, you need a [Datadog account][1].
+
+To run the example application, you need a machine with [GNU Make][2] and [Docker][3]. Have your Datadog [API key][4] available.
+
+### Install the example application
+
+The example application starts up the Datadog Agent and a PostgreSQL database in a Docker container. While the application runs, the Agent sends database metrics to Datadog. You can view the data from the application in Datadog Database Monitoring.
+
+Follow these instructions to install the example application on MacOS or Linux.
+
+1. Download the [archive][5] containing the example application.
+
+2. Extract the archive to a location on disk:
+```
+tar -xvf database-management.tgz
+```
+
+3. Change to the `database-management` directory in the extracted archive:
```

Comment:
Where does `database-management` come from? This could be confusing since the name of the product is `database monitoring`.

Reply:
oops, thank you for catching this! `database-management` is a typo on my part.